scene/gui/control.cpp

```diff
@@ -2358,6 +2358,19 @@ void Control::_window_find_focus_neighbor(const Vector2 &p_dir, Node *p_at, cons
 		points[2] = xform.xform(c->get_size());
 		points[3] = xform.xform(Point2(0, c->get_size().y));
 
+		// Tiebreaking variables
```

Comment:
Add a more descriptive comment:

```suggestion
		// Tie-breaking refers to situations where a potential focus neighbor's bounding rect
		// is right next to the currently focused control (e.g. in BoxContainer with
		// separation overridden to 0). This needs specific handling so that the correct
		// focus neighbor is selected.
```